src/WebExploitation/FileUploads/Insufficient blacklisting.md

```diff
@@ -0,0 +1,88 @@
+# File Uploads: Insufficient blacklisting
+
+## Introduction
+In the wild, it's unlikely that you'll find a website that has no protection against file upload attacks like we saw in the first lab. But just because defenses are in place, that doesn't mean that they're robust. You can sometimes still exploit flaws in these mechanisms to obtain a web shell for remote code execution. 
```

Comment:
This section is repeating across all aritcles, it'd be nice to have a unique intro for each topic